docs/sources/explore/correlations-editor-in-explore.md

```diff
@@ -0,0 +1,118 @@
+1. Explore opens a split view. Use the right pane to setup [the target query source of the correlation]({{< relref "../administration/correlations/correlation-configuration#target-query" >}}).
+1. Build target query using [variables syntax]({{< relref "../dashboards/variables/variable-syntax" >}}) with variables from the list provided at the top of the pane. The list contains sample values from the selected data row.
+1. Provide optional label and description. Label is used as the name of the link inside the visualization. It can contain variables.
+1. Click "Save" button in the top bar to save the correlation and exit Correlations Editor mode.
+1. The link used to create the correlation will be replaced with a data link in each row. The variables will be dynamically replaced with values from a selected row.
+
+## Transformations
+
+Transformations allow us to extract values that exist in a field with other data. A prime example is getting one portion of a log line out to use in a correlation. For more details on transformations in correlations, see the [correlations documentation]({{< relref "../administration/correlations/correlation-configuration/#correlation-transformations" >}}).
+
+After clicking one of the generated links in the editor mode, you can add transformations by clicking the "Add transformation" button in the "Transformations" dropdown. This will pop up a modal, and you can use a transformation in your correlation with the following steps:
+
+1. First, select a field that the transformation will be applied. This should be a field where we only want to use sections of the field, such as a log line. Once selected, the value of that field will show up to assist building the transformation.
+1. Select the type of the transformation. See the [correlations documentation]({{< relref "../administration/correlations/correlation-configuration/#correlation-transformations" >}}) for the options and relevant settings.
+1. Based on what's selected, you may immediately see one or more variables populate, or you may need to fill in more fields
+1. Select `Add transformation to correlation` to add the specified variables to the list of variables in the list of variables available.
+
+### Notes for regular expressions
+
+For regular expressions in this modal, the `mapValue` referred to in other documentation is called `Variable Name` here. The field value will highlight with the matching text based on the expression. Use regular expression capture groups to select what portion of the match should be extracted. When a valid regular expression is provided, you will see the variable and the value of that variable appear below the `Variable Name` field.
```

Comment:
I'm not sure I understand "For regular expressions in this modal, the `mapValue` referred to in other documentation is called `Variable Name` here"
```suggestion
For regular expressions in this dialog box, the `mapValue` referred to in other documentation is called `Variable Name` here.
The field value highlights the matching text based on the expression.
Use regular expression capture groups to select what portion of the match should be extracted.
When a valid regular expression is provided, the variable and the value of that variable appear below the `Variable Name` field.
```

Reply:
> I'm not sure I understand "For regular expressions in this modal, the mapValue referred to in other documentation is called Variable Name here"

MapValue is the original term we used in the provisioning and original correlations editor. With this editor being more user friendly in what it was doing, we decided to use a more user friendly name. See https://github.com/grafana/grafana/blob/main/docs/sources/administration/correlations/correlation-configuration/index.md?plain=1#L99 for the use of mapValue in other documentation

Reply:
Changing
>  The field value will highlight with the matching text based on the expression

to 

> The field value highlights the matching text based on the expression.

Doesn't quite make sense in my opinion. As you type an expression, a highlight shows according to the match on the expression. The field value is highlighted, it doesn't do the highlighting.

https://github.com/grafana/grafana/assets/1533128/03199793-dd23-4b71-b9a5-a2cc1b412a62

Maybe "Any text that matches the expression in the field value is highlighted." ?

Reply:
I see what you are saying but I don't think my suggestion changes the existing meaning.
The original text "The field value will highlight" also suggests that the field value is doing the highlighting.

I think your rewrite is more technically accurate but is in passive voice. In general we try and prefer active voice but it is harder to rewrite this actively because I'm not actually sure what thing does the highlighting. Perhaps:

> Grafana highlights any text that matches the expression in the field value.